frontend/src/app/api/addToSheet/route.ts

```diff
@@ -0,0 +1,45 @@
+import { NextResponse } from 'next/server';
+import { google } from 'googleapis';
+import { formatInTimeZone } from 'date-fns-tz';
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    const { name, email, role } = body;
+
+    // Format the current date and time in desired timezone as yyyy/MM/dd HH:mm:ss
+    const createdAt = formatInTimeZone(
+      new Date(),
+      process.env.WAITLIST_TIMEZONE || 'UTC',
+      'yyyy/MM/dd HH:mm:ss'
+    );
+
+    console.log(createdAt);
+
+    const auth = new google.auth.GoogleAuth({
+      credentials: {
+        client_email: process.env.GOOGLE_CLIENT_EMAIL,
+        private_key: process.env.GOOGLE_PRIVATE_KEY?.replace(/\\n/g, '\n'),
+      },
+      scopes: ['https://www.googleapis.com/auth/spreadsheets'],
+    });
+
+    const sheets = google.sheets({ version: 'v4', auth });
+
+    const spreadsheetId = process.env.SPREADSHEET_ID;
+
+    const response = await sheets.spreadsheets.values.append({
+      spreadsheetId,
+      range: 'Sheet1!A:C',
+      valueInputOption: 'USER_ENTERED',
+      requestBody: {
+        values: [[name, email, role, createdAt]],
```

Comment:
i created some submisions and the usertpe was omitted. this is actually a required submission value